### ISSUE TITLE
Report abuse

### ISSUE THREAD

**Issue by armantorkzaban** [tcfev/forDem#98]

**Comment by desputinski** [comment]
I think this is probably also linked to moderation, correct?  Do we already have an issue about moderation functionalities in the backend? (i.g. also including finding forbidden/illegal words)

**Comment by armantorkzaban** [comment]
Yes, it is connected to moderation and is now listed under a label with the same name.
